var/spack/repos/builtin/packages/pthreadpool/package.py

```diff
@@ -50,6 +50,7 @@ class Pthreadpool(CMakePackage):
 
     def cmake_args(self):
         return [
+            '-DBUILD_SHARED_LIBS=ON',
```

Comment:
```suggestion
            self.define('BUILD_SHARED_LIBS', True),
```
CMakePackage has a helper function for this.

Reply:
I noticed that, but it looked like 99% of other packages doing this weren't using it, so I assumed it was only for dynamic values. Happy to switch if that's preferred.